source/verify-packages.txt

```diff
@@ -0,0 +1,244 @@
+   :style: normal
+
+   .. step:: Download the {+atlas-cli+} installation file.
+
+      Download the {+atlas-cli+} binaries from the 
+      `MongoDB Download Center
+      <https://www.mongodb.com/try/download/atlascli>`__
+      based on your Linux environment. Click :guilabel:`Copy link` and
+      use the URL in the following instructions.
+
+      For example, to download the ``{+atlas-cli-version+}`` release
+      for Linux through the shell, run this command:
+
+      .. code-block::
+
+         curl -LO https://fastdl.mongodb.org/mongocli/mongodb-atlas-cli_{+atlas-cli-version+}_linux_x86_64.tar.gz
+
+   .. step:: Download the public signature file.
+
+      Run the following command to download the file:
+
+      .. code-block::
+
+         curl -LO https://fastdl.mongodb.org/mongocli/mongodb-atlas-cli_{+atlas-cli-version+}_linux_x86_64.tar.gz.sig
+
+   .. step:: Download and import the key file.
+
+      Run the following command to download and import the key file:
+
+      .. io-code-block::
+         :copyable: true
+
+         .. input::
+
+            curl -LO https://pgp.mongodb.com/atlas-cli.asc
+            gpg --import atlas-cli.asc
+            
+         .. output::
+
+            gpg: key <key-value-short>: public key "Atlas CLI Release Signing Key <packaging@mongodb.com>" imported
+            gpg: Total number processed: 1
+            gpg:               imported: 1
+
+   .. step:: Verify the {+atlas-cli+} installation file.
+
+      Run the following command to verify the installation file:
+
+      .. io-code-block::
+         :copyable: true
+
+         .. input::
+
+            gpg --verify mongodb-atlas-cli_{+atlas-cli-version+}_linux_x86_64.tar.gz.sig mongodb-atlas-cli_{+atlas-cli-version+}_linux_x86_64.tar.gz
+            
+         .. output::
+
+            gpg: Signature made Thu Mar 14 08:25:00 2024 EDT
+            gpg:                using RSA key <key-value-long>
+            gpg: Good signature from "Atlas CLI Release Signing Key <packaging@mongodb.com>" [unknown]
+
+      If the package is properly signed, but you do not currently trust
+      the signing key, ``gpg`` also returns the following message :
+
+      .. code-block::
+
+         gpg: WARNING: This key is not certified with a trusted signature!
+         gpg:          There is no indication that the signature belongs to the owner.
+
+.. _verify-packages-windows:
+
+Verify Windows Packages
+-----------------------
+
+The following procedure verifies the {+atlas-cli+} package against its 
+SHA-256 key.
+
+.. procedure::
+   :style: normal
+
+   .. step:: Download the {+atlas-cli+} installation file.
+
+      Download the {+atlas-cli+} ``.msi`` or ``.zip`` file from the 
+      `MongoDB Download Center
+      <https://www.mongodb.com/try/download/atlascli>`__ or 
+      `Github <https://github.com/mongodb/mongodb-atlas-cli/releases>`__.
+
+
+
+   .. step:: Save the public signature.
+
+      a. Open the ``checksums.txt`` file for the
```

Comment:
Maybe download the checksums file for the release should be the first step and open should be second.